### PR TITLE
Update org-wide-files

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- ci: update configlet workflow
- ci: remove trailing whitespace from labels.yml
- Update license to 2022

Having updated these files, new repos won't immediately get a pull request to update these files.

### CODE REVIEW

**Comment by kytrinyx** [comment]
Is this change because of the rate limits that we were hitting on GitHub Actions? If so, how does this change affect the rate limits, and when do the repos get the PRs for the changes?

**Comment by ErikSchierboom** [comment]
> Is this change because of the rate limits that we were hitting on GitHub Actions?

Not directly, no. This change is just there to prevent the org-wide-files PR from being created in the first place, as the files in this repo were outdated when compared to the ones in the org-wide-files repo.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Exercism
+Copyright (c) 2022 Exercism
```
Comment (ErikSchierboom) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 Exercism
```

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply LICENSE]: We should probably stop the license.md file from being synced I think. But for now, I've reverted.

**Pull request merged**